src/Parts/Component.php

```diff
@@ -23,11 +23,11 @@ class Component
     public ?string $url;
     public ?bool $disabled;
     /**
-     * @var \Ragnarok\Fenrir\Parts\ComponentSelectOptions[]
+     * @var ComponentSelectOptions[]
      */
     public ?array $options; // @todo
     /**
-     * @var \Ragnarok\Fenrir\Enums\Parts\ChannelTypes[]
+     * @var ChannelTypes[]
      */
```

Comment:
3rd party json mapper requires complete paths in doc blocks (not blaming you for not knowing about this limitation)